src/ydata_synthetic/synthesizers/regular/cgan/model.py

```diff
@@ -160,7 +160,7 @@ def build_model(self, input_shape, dim):
 
 if __name__ == '__main__':
     import pandas as pd
-    from preprocessing.credit_fraud import transformations
+    from src.ydata_synthetic.preprocessing import transformations
```

Comment:
this will be a problem when installing in the client, right? we already saw that in the ydata lib.

btw, why are you using `src` if we talk and "decide" (i guess) to drop it in the ydata lib?

Reply:
Already corrected!